app/assets/javascripts/admin/transaction_modal_view.coffee

```diff
@@ -10,13 +10,13 @@ class MS.Views.TransactionModalView extends Backbone.View
   initialize: (params) ->
     @loanId = params.loanId
     @locale = params.locale
-    url = "/admin/accounting/transactions/new"
-    @loadContent(url, @loanId, 'new')
```

Comment:
this is called thru constructor even if action is 'show,' kicking off race with the other correct call to loadContent in 'show'